Tao_Current.txt

```diff
@@ -569,63 +641,67 @@ Considering all this, it's not surprising that some IETFers would prefer to have
+[BCP101](https://tools.ietf.org/html/rfc4071) Austein, R. and B. Wijnen, “Structure of the IETF Administrative Support Activity (IASA)”, BCP 101, RFC 4071, April 2005.
 
-[BCP102] Daigle, L. and Internet Architecture Board, “IAB Processes for Management of IETF Liaison Relationships”, BCP 102, RFC 4052, April 2005.
+[BCP102](https://tools.ietf.org/html/rfc4052) Daigle, L. and Internet Architecture Board, “IAB Processes for Management of IETF Liaison Relationships”, BCP 102, RFC 4052, April 2005.
 
-[BCP103] Trowbridge, S., Bradner, S., and F. Baker, “Procedures for Handling Liaison Statements to and from the IETF”, BCP 103, RFC 4053, April 2005
+[BCP103](https://tools.ietf.org/html/rfc4053) Trowbridge, S., Bradner, S., and F. Baker, “Procedures for Handling Liaison Statements to and from the IETF”, BCP 103, RFC 4053, April 2005
 
-[RFC1796] Huitema, C., Postel, J., and S. Crocker, “Not All RFCs are Standards”, RFC 1796, April 1995.
+[RFC1796](https://tools.ietf.org/html/rfc1796) Huitema, C., Postel, J., and S. Crocker, “Not All RFCs are Standards”, RFC 1796, April 1995.
 
-[RFC2223] Postel, J. and J. Reynolds, “Instructions to RFC Authors”, RFC 2223, October 1997.
+[RFC2223](https://tools.ietf.org/html/rfc2223) Postel, J. and J. Reynolds, “Instructions to RFC Authors”, RFC 2223, October 1997.
 
-[RFC2860] Carpenter, B., Baker, F., and M. Roberts, “Memorandum of Understanding Concerning the Technical Work of the Internet Assigned Numbers Authority”, RFC 2860, June 2000.
+[RFC2860](https://tools.ietf.org/html/rfc2860) Carpenter, B., Baker, F., and M. Roberts, “Memorandum of Understanding Concerning the Technical Work of the Internet Assigned Numbers Authority”, RFC 2860, June 2000.
 
-[RFC6635] Kolkman, O. and J. Halpern, “RFC Editor Model (Version 2)”, RFC 6635, March 2012.
+[RFC6635](https://tools.ietf.org/html/rfc6635) Kolkman, O. and J. Halpern, “RFC Editor Model (Version 2)”, RFC 6635, March 2012.
 
-[RFC4677] Hoffman, P. and S. Harris, “The Tao of IETF - A Novice's Guide to the Internet Engineering Task Force”, RFC 4677, September 2006.
+[RFC4677](https://tools.ietf.org/html/rfc4677) Hoffman, P. and S. Harris, “The Tao of IETF - A Novice's Guide to the Internet Engineering Task Force”, RFC 4677, September 2006.
 
-[RFC4858] Levkowetz, H., Meyer, D., Eggert, L., and A. Mankin, “Document Shepherding from Working Group Last Call to Publication”, RFC 4858, May 2007.
+[RFC4858](https://tools.ietf.org/html/rfc4858) Levkowetz, H., Meyer, D., Eggert, L., and A. Mankin, “Document Shepherding from Working Group Last Call to Publication”, RFC 4858, May 2007.
 
-[RFC6722] Hoffman, P., “Publishing the "Tao of the IETF" as a Web Page”, RFC 6722, August 2012.
+[RFC6722](https://tools.ietf.org/html/rfc6722) Hoffman, P., “Publishing the "Tao of the IETF" as a Web Page”, RFC 6722, August 2012.
+
+[STD3](https://tools.ietf.org/html/rfc1123) Braden, R., “Requirements for Internet Hosts - Application and Support”, STD 3, RFC 1123, October 1989.
```

Comment:
STD 3 is RFC 1122 _and_ RFC 1123. It's probably best to link to https://tools.ietf.org/html/std3 instead of a specific RFC.

Reply:
Accepted and changed